docs/userguide/constraints.md

```diff
@@ -222,10 +233,123 @@ DiscreteLinkedParametersConstraint(
 | 3 | Octanol           | Octanol             |                                         |
 
 ### ``DiscreteDependenciesConstraint``
-Content coming soon...
+A dependency is a situation where parameters depend on other parameters.
+Let's say an experimental setup has a parameter called "Switch" which turns on 
+pieces of equipment that are optional.
+This means the other parameters (called ``affected_parameters``) are only relevant if 
+the "Switch" parameter has the value "on". If the switch is "off", the affected parameters 
+are irrelevant.
+
+You can specify such a dependency with the ``DiscreteDependenciesConstraint``, which 
+requires:
+1) A list ``parameters`` with the names of the parameters upon which others depend.
+2) A list of ``Condition``'s which specify the values of the corresponding entries in 
```

Comment:
Rather "`Condition` objects", that looks nicer imo

Reply:
changed as per other comment